src/utils/ChoicesWrapper.js

```diff
@@ -68,7 +68,7 @@ class ChoicesWrapper extends Choices {
       return;
     }
 
-    this.shouldOpenDropDown = false;
+    this.shouldOpenDropDown = true;
```

Comment:
This change claims that this variable is not needed. You may as well just delete all references to it since it checks to see if it is false later and then sets it to true. I can't imagine that this was added for nothing. I would need to consult with the developer who introduced this to see if this is alright to remove.

Reply:
Let me know if it's ok to remove this. If yes, I will proceed with the change you suggested :v:

Reply:
Hey. Do you have any news on this MR? Should I close this PR?

Reply:
I have a hard time believing we would have added this ```shouldOpenDropDown``` flag for nothing. Your change pretty much rips it out, so I do have some concerns that this will break the original need for it. Go ahead and change this pull request to remove this entirely and we may pull this into a test build to try it out.